services/llm-provider/src/config/settings.py

```
@@ -68,13 +70,28 @@ class Settings:
+    @property
+    def requires_policy_gateway_mediation(self) -> bool:
+        """Return True when direct provider calls must fail closed."""
+        return self.release_gate or self.environment.lower() in {
+            "production",
+            "prod",
+            "staging",
+            "stage",
+        }
+
     @classmethod
     def from_env(cls) -> "Settings":
```

Ruff (0.15.12)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)